code/iaas/auth-logic/src/main/java/io/cattle/platform/iaas/api/auth/dao/impl/AuthDaoImpl.java

```diff
@@ -304,8 +305,8 @@ public boolean isProjectMember (long projectId, Long usingAccount, boolean isAdm
                 for (ProjectMember member : previousMembers) {
                     otherPreviosMembers.add(new Member(member));
                 }
-                HashSet<Member> create = (HashSet<Member>) ((HashSet<Member>) members).clone();
-                HashSet<Member> delete = (HashSet<Member>) ((HashSet<Member>) otherPreviosMembers).clone();
+                Set<Member> create = new HashSet<Member>(members);
+                Set<Member> delete = new HashSet<Member>(otherPreviosMembers);
```

Comment:
@hibooboo2 Was there a specific reason for clone vs the copy constructor here? Just want to make sure I'm not breaking something you did intentionally.